Cargo.toml

```diff
@@ -108,6 +108,7 @@ uuid = { version = "1.15.1", default-features = false, features = ["v4", "fast-r
 jsonschema = { version = "0.30.0", default-features = false, optional = true }
 chrono = { version = "0.4.40", optional = true }
 chrono-tz = { version = "0.10.1", optional = true }
+ipnet = { version = "2.11.0", optional = true }
```

Comment:
Worth exploring `default-features = false`. That will allow use in no_std environments as well.

Reply:
@anakrish thanks for the pointer. I'm a little unsure if there's anything more I need to do to ensure this builtin works in `no_std` environments other than adding `default-features = false`. I see that there is an `opa-no-std` featureset defined, but not all of the dependencies marked `default-features = false` are included there. Could you help me understand if there's anything else I should do?

Reply:
There is nothing  more that needs to be done other than adding it to the `opa-no-std` list above. 
I see that ipnet has an `std` feature which implies that it has `no_std` support.